randovania/games/samus_returns/json_data/Area 2 - Exterior.txt

```diff
@@ -404,7 +404,7 @@ Extra - asset_id: collision_camera_024
   > Tunnel to Outer Hub Shaft (Top)
       All of the following:
           Lay Any Bomb
-          Single-wall Wall Jump (Beginner) or Climb Rooms Vertically (No High Jump)
+          Space Jump or Single-wall Wall Jump (Beginner) or Simple IBJ
```

Comment:
does spider not work here?
![grafik](https://github.com/randovania/randovania/assets/38186597/59203976-11e2-4d97-b262-e46b393ac854)


on further looking, i think you may have selected the worng node. because the lower tunnel to the aeion tank still has the template even tho spider doesnt work there
(also spacejump should probably be limited by some movement there too)

Reply:
Ah I did change the wrong node.